sigma/backends/netwitness/netwitness.py

```diff
@@ -65,16 +70,18 @@ class NetWitnessBackend(TextQueryBackend):
     field_quote: ClassVar[Optional[str]] = None
     # Quote field names if this pattern (doesn't) matches, depending on field_quote_pattern_negation.
     # Field name is always quoted if pattern is not set.
-    field_quote_pattern: ClassVar[Optional[Pattern[str]]] = re.compile("^\\w+$")
+    field_quote_pattern: ClassVar[Optional[Pattern[str]]] = None
     # Negate field_quote_pattern result. Field name is quoted if pattern doesn't matches if set to True (default).
     field_quote_pattern_negation: ClassVar[bool] = True
```

Comment:
I would personally sort the lines of code in a different order. I would put all comments to the top and after this the three properties controlling the field qoutation.